### PR TITLE
[Doc]Update the supported data types of bitmap and bloomfilter indexes

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

File: docs/en/table_design/indexes/Bitmap_index.md [hunk 1]
```diff
@@ -29,7 +29,11 @@ You can benefit from bitmap indexes in the following aspects:
 
 - You can create a bitmap index for a column that can be filtered by using the equal (`=`) or [NOT] IN operator.
 - You can create bitmap indexes for all columns of a Duplicate Key table or Unique Key table. For an Aggregate table or Primary Key table, you can only create bitmap indexes for key columns.
-- The columns of the FLOAT, DOUBLE, BOOLEAN, and DECIMAL types do not support creating bitmap indexes.
+- Bitmap indexes can be created for the columns of the following  types:
+    - Numeric types: TINYINT, SMALLINT, INT, BITGINT, LARGEINT, DECIMAL, and BOOLEAN.
+    - String types: CHAR, STRING, and VARCHAR.
+    - Date types: DATE, DATETIME.
```
Comment (amber-create) [inline docs/en/table_design/indexes/Bitmap_index.md:35]: DATE, DATETIME. -> DATE and DATETIME.

File: docs/en/table_design/indexes/Bloomfilter_index.md [hunk 2]
```diff
@@ -20,7 +20,10 @@ For example, you create a bloom filter index on a `column1` of a given table `ta
 ## Usage notes
 
 - You can create bloom filter indexes for all columns of a Duplicate Key or Primary Key table. For an Aggregate table or Unique Key table, you can only create bloom filter indexes for key columns.
-- TINYINT, FLOAT, DOUBLE, and DECIMAL columns do not support creating bloom filter indexes.
+- The Bloom filter indexes can be created for the columns of the following types:
```
Comment (amber-create) [inline docs/en/table_design/indexes/Bloomfilter_index.md:23]: image
remove "The"

File: docs/en/table_design/indexes/Bitmap_index.md [hunk 3]
```diff
@@ -29,7 +29,11 @@ You can benefit from bitmap indexes in the following aspects:
 
 - You can create a bitmap index for a column that can be filtered by using the equal (`=`) or [NOT] IN operator.
 - You can create bitmap indexes for all columns of a Duplicate Key table or Unique Key table. For an Aggregate table or Primary Key table, you can only create bitmap indexes for key columns.
-- The columns of the FLOAT, DOUBLE, BOOLEAN, and DECIMAL types do not support creating bitmap indexes.
+- Bitmap indexes can be created for the columns of the following  types:
```
Comment (amber-create) [inline docs/en/table_design/indexes/Bitmap_index.md:32]: image
1. remove "the"
2. extra space
3. types -> data types

File: docs/en/table_design/indexes/Bloomfilter_index.md [hunk 4]
```diff
@@ -20,7 +20,10 @@ For example, you create a bloom filter index on a `column1` of a given table `ta
...
+    - Numeric types: SMALLINT, INT, BIGINT, and LARGEINT.
+    - String types: CHAR, STRING, and VARCHAR.
+    - Date types: DATE, DATETIME.
```
Comment (amber-create) [inline docs/en/table_design/indexes/Bloomfilter_index.md:26]: see above comment

**Review by amber-create — DISMISSED** [review]

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**